java/androidpayload/library/src/com/metasploit/meterpreter/AndroidMeterpreter.java

```diff
@@ -44,6 +44,8 @@
 import com.metasploit.meterpreter.stdapi.stdapi_net_config_get_routes_V1_4;
 import com.metasploit.meterpreter.stdapi.stdapi_net_socket_tcp_shutdown_V1_3;
 import com.metasploit.meterpreter.stdapi.stdapi_sys_process_execute_V1_3;
+//Umerov Artem
```

Comment:
If this comment is really necessary (your name will be in the commit history anyway) then it might worth moving it to the set_audio_mode_android.java class.